packages/patternfly-3/patternfly-react/src/components/DateTimePicker/DateComponents/Day.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { isEquelDate, isWeekend } from './helpers';
+
+class Day extends React.Component {
+  onClick = e => {
+    e.preventDefault();
+    this.props.setSelected(this.props.day);
+  };
+  render() {
+    const day = this.props.day.getDate();
+    const { day: date, currDate, selectedDate } = this.props;
+
+    const now = new Date();
+    const className = classNames('day', {
+      weekend: isWeekend(date),
+      old: date.getMonth() !== currDate.getMonth(),
+      active: isEquelDate(date, selectedDate),
+      today: isEquelDate(date, now)
+    });
```

Comment:
optional: could this component receive all of the props ready from its parent,
instead of calculation everything here?